packages/calling-component-bindings/src/handlers/createCommonHandlers.ts

```diff
@@ -121,7 +128,10 @@ export const createDefaultCommonCallingHandlers = memoizeOne(
   (
     callClient: StatefulCallClient,
     deviceManager: StatefulDeviceManager | undefined,
-    call: Call | /* @conditional-compile-remove(teams-identity-support) */ TeamsCall | undefined
+    call: Call | /* @conditional-compile-remove(teams-identity-support) */ TeamsCall | undefined,
+    options?: {
+      onResolveVideoBackgroundDependency?: () => Promise<VideoEffectBackgroundDependency>;
```

Comment:
nit: inconsistent here between `VideoBackgroundDependency` and `VideoEffectBackgroundDependency`

I assume all should be `video effects background dependency` (in relevant camel/Pascal Case, note the plural effects not effect)